server.js

```diff
@@ -0,0 +1,59 @@
+'use strict';
+//application setup
+var express = require('express');
+var path = require('path');
+var app = express();
+var port = 3000;
+
+//view directory setup
+app.set('views', path.join(__dirname, 'views'));
+//view engine setup
+app.set('view engine', 'jade');
+
+//index get requests
+app.get('/', function(req, res){
+  res.render('index');
+});
+app.get('/about', function(req, res){
+  res.render('about');
+});
+app.get('/footer', function(req, res){
+  res.render('footer');
+});
+app.get('/head', function(req, res){
+  res.render('head');
+});
+app.get('/nav', function(req, res){
+  res.render('nav');
+});
+app.get('/users', function(req, res){
+  res.render('users');
+});
```

Comment:
We were working on being modular. So we render our page from modules and include them on the larger pages.

Kristopher Skelton
www.linkedin.com/in/kristopherskelton/


Date: Wed, 8 Jul 2015 12:48:55 -0700
From: notifications@github.com
To: express-yourself@noreply.github.com
Subject: Re: [express-yourself] Templating (#2)

In server.js:

> +});
> +app.get('/about', function(req, res){
> +  res.render('about');
> +});
> +app.get('/footer', function(req, res){
> +  res.render('footer');
> +});
> +app.get('/head', function(req, res){
> +  res.render('head');
> +});
> +app.get('/nav', function(req, res){
> +  res.render('nav');
> +});
> +app.get('/users', function(req, res){
> +  res.render('users');
> +});

Why do you have routes for specific sections of your page? Was this for testing?


—
Reply to this email directly or view it on GitHub.